pkg/descheduler/node/node.go

```diff
@@ -77,7 +77,7 @@ func IsReady(node *v1.Node) bool {
 		// - NodeOutOfDisk condition status is ConditionFalse,
 		// - NodeNetworkUnavailable condition status is ConditionFalse.
 		if cond.Type == v1.NodeReady && cond.Status != v1.ConditionTrue {
-			klog.V(1).Infof("Ignoring node %v with %v condition status %v", node.Name, cond.Type, cond.Status)
+			klog.V(1).InfoS("Ignoring node", "node", klog.KObj(node), "condition", cond.Type, "status", cond.Status)
```

Comment:
```suggestion
			klog.V(1).InfoS("ignoring node", "node", klog.KObj(node), "condition", cond.Type, "status", cond.Status)
```

Reply:
nit: nodes are not namespaced so node.Name will do as well